README.md

```diff
@@ -31,7 +30,7 @@ The key of each entry has to be unique and will be used for the config file name
   become: true
   vars:
     logstash_config_files:
-      - name: syslog
+	  syslog:
```

Comment:
The key is further indented than it's children.

Reply:
fixed